File: android/app/src/main/kotlin/com/example/hotel_reception/procesor/staff/HotelReception.kt
package com.example.hotel_reception.procesor.staff

import com.example.hotel_reception.procesor.model.Person

interface HotelReception {
    fun checkoutCustomers()
    fun checkInPersons(persons: List<Person>)
    fun checkRoomsNeedRepairing()
    fun renewRoomsAfterRepairing()
}

File: ios/Runner/Processor/staff/HotelReception.swift
//
//  HotelReception.swift
//  Runner
//
//  Created by Serhii Rakovskyi on 18.08.2022.
//

import Foundation

protocol HotelReception {
    func checkoutCustomers()
    func checkInPersons(persons: Array<Person>)
    func checkRoomsNeedRepairing()
    func renewRoomsAfterRepairing()
}
